fiftyone/operators/types.py

```diff
@@ -328,12 +328,10 @@ def img(
 
             import fiftyone.operators.types as types
 
-            ctx.panel.state.my_img = "https://i.imgur.com/TGEZj1Rl.jpg"
-
             panel = types.Object()
             panel.img(
                 "my_img",
-                href="https://voxel51.com",
+                href="https://path/to/my/image",
```

Comment:
I believe the previous doc was correct. This is not the path to the image. Instead this property defines a string which is a path to an image.